ci-operator/step-registry/additional-enabled-capabilities/additional-enabled-capabilities-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Sounds like the script is updating install-config.yaml, if that, maybe use ipi/conf  as parent folder maybe more proper.

Reply:
Okay, the same comments as https://github.com/openshift/release/pull/30855/files#r931896434

Reply:
good suggestion, will move to ipi/conf folder.